go/libkb/globals.go

```diff
@@ -226,7 +228,7 @@ func (g *GlobalContext) createLoginStateLocked() {
 		g.loginState.Shutdown()
 	}
 	g.loginState = NewLoginState(g)
-	g.ActiveDevice = new(ActiveDevice)
+	g.ActiveDevice.clear(g.loginState.account)
```

Comment:
Well... Then it has the chance to fail if the Account queue is stuck. I don't want to make things worse by giving `Logout` the opportunity to accidentally not clear the ActiveDevice. Because that's the foundation of keeping future users of a process out of the logged-out user's secrets (admittedly a fuzzy concept).